test/functional/feature_coinstatsindex.py

```diff
@@ -226,7 +228,7 @@ def _test_coin_stats_index(self):
 
         self.generate(index_node, 1, sync_fun=self.no_op)
         res10 = index_node.gettxoutsetinfo('muhash')
-        assert res8['txouts'] < res10['txouts']
+        assert_greater_than(res10['txouts'], res8['txouts'])
```

Comment:
I noticed that here (and at some other places), `assert x < y` is replaced by `assert_greater_than(y, x)` rather than the more obvious `assert_less_than(x, y)`. Is that change intentional or is that a left-over from an earlier version, when `assert_left_than` was not introduced yet?

Reply:
Yes, early on I kept only `assert_greater_than()`, before I decided that `assert_less_than()`was needed. I can go back and fix those to make it clear.

Reply:
Done. Force pushed to allow for the rebase.